app/Components/Background/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ImageBackground, View, ViewStyle } from 'react-native';
+import styles from './style';
+
+interface Props {
+    children: React.ReactNode;
+}
+
+const CommonBackground = ({ children }: Props) => {
+    return (
+        <ImageBackground
+            source={require('../../Assets/images/background.jpg')}
```

Comment:
Store images in a separate file and then import them into the component.
eg: const background = require('../../Assets/images/background.jpg')

component - import {background} from 'constant file path'